qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -36,30 +36,23 @@ class DrivenControl:
 
     Parameters
     ----------
-    rabi_rates : np.ndarray, optional
+    rabi_rates : np.ndarray
```

Comment:
Hmm, unlike a lot of the other default values that you're (rightfully) getting rid of, I actually think `rabi_rates`, `azimuthal_angles`, and `detunings` defaulting to zero actually makes some sense. It would be quite common to have a control that only sets `rabi_rates`, for example, and in that case it's nice to save the user the trouble of creating zeros manually.

I agree that `durations` shouldn't have a default though.

Reply:
Good point. Now only remove default values for `detunings.

We actually use `pi` as the default value for `rabi_rates` when all inputs are `None` and this should be fixed now as  `duration` is not allowed to be `None`.

Also, add the test back.